test/e2e/performanceprofile/functests/utils/tuned/tuned.go

```diff
@@ -3,6 +3,7 @@ package tuned
 import (
 	"context"
 	"fmt"
+	"hash/fnv"
```

Comment:
I'm not against this approach, but there should be another PR which adds the bindata compilation anyway (#1019 ?). Let's sync about this.

Reply:
Good. I went with the opposite approach but probably in your case it was a must.
I suggest to apply https://github.com/openshift/cluster-node-tuning-operator/pull/1019/commits/426451af8e24552b16c1c339ddbf04b6469f766d on separate PR and the sooner the better.